cmake/sysbuild/suit.cmake

```diff
@@ -261,6 +261,38 @@ function(suit_create_package)
     )
   endforeach()
 
+  # First parse which images should be extracted to which cache partition
+  set(DFU_CACHE_PARTITIONS_USED "")
+  foreach(image ${IMAGES})
+    sysbuild_get(EXTRACT_TO_CACHE IMAGE ${image} VAR CONFIG_SUIT_EXTRACT_IMAGE_TO_DFU_CACHE KCONFIG)
+    if(EXTRACT_TO_CACHE)
+      sysbuild_get(CACHE_PARTITION_NUM IMAGE ${image} VAR CONFIG_SUIT_IMAGE_DFU_CACHE_PARTITION KCONFIG)
+      list(APPEND DFU_CACHE_PARTITIONS_USED ${CACHE_PARTITION_NUM})
+      list(APPEND SUIT_CACHE_PARTITION_${CACHE_PARTITION_NUM} ${image})
+    endif()
+  endforeach()
+  list(REMOVE_DUPLICATES DFU_CACHE_PARTITIONS_USED)
+
+  # Then create the cache partitions
+  foreach(CACHE_PARTITION_NUM ${DFU_CACHE_PARTITIONS_USED})
+    set(CACHE_CREATE_ARGS "")
+    foreach(image ${SUIT_CACHE_PARTITION_${CACHE_PARTITION_NUM}})
+      sysbuild_get(BINARY_DIR IMAGE ${image} VAR APPLICATION_BINARY_DIR CACHE)
+      sysbuild_get(BINARY_FILE IMAGE ${image} VAR CONFIG_KERNEL_BIN_NAME KCONFIG)
+      sysbuild_get(IMAGE_CACHE_URI IMAGE ${image} VAR CONFIG_SUIT_IMAGE_DFU_CACHE_URI KCONFIG)
+      list(APPEND CACHE_CREATE_ARGS
+        "--input" "\"${IMAGE_CACHE_URI},${BINARY_DIR}/zephyr/${BINARY_FILE}.bin\""
+      )
+    endforeach()
+    list(APPEND CACHE_CREATE_ARGS "--output-file" "${SUIT_ROOT_DIRECTORY}dfu_cache_partition_${CACHE_PARTITION_NUM}.bin")
+
+    if (SUIT_DFU_CACHE_PARTITION_${CACHE_PARTITION_NUM}_EB_SIZE)
```

Comment:
uniform style, no space between if and bracket

Reply:
Fixed